### DISCUSSION TITLE
Publish tag documentation not working

### DISCUSSION THREAD

**Discussion by masakik** [spatie/laravel-activitylog#883] [Q&A] [answered]
In commit ee97e07 the publish tag was changed to "--tag=activitylog-config" but it is not working for me.
It worked using the old way, --tag=config. Same for migrations.
I am using laravel 8.40.0.
Am I missing something?

**Comment by Gummibeer** [comment]
Thanks for reporting - I was in hope that the adjustments were tested. But now I will spin up a fresh Laravel instance and test everything my own ...

**Reply by Gummibeer** [reply]
Hey,
are you sure that you are on v4 of that package? It requires Laravel 8 and PHP 8.
In case you are: the adjustments in the mentioned PR are correct.
In case you are still on v3 you should also check the v3 docs: https://spatie.be/docs/laravel-activitylog/v3/installation-and-setup

**Comment by masakik** [comment, marked as answer]
Sorry, I'm using version 3 with php 7.4.
